### ISSUE TITLE
Skiplink container fails WAVE accessibility scan

### ISSUE THREAD

**Issue by pwright08** [alphagov/govuk_template#295]
The skiplink container fails the [WAVE Evaluation Tool](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) accessibility scan because of low contrast.

A way to fix this would be to make the [skiplink container](https://github.com/alphagov/govuk_template/blob/5e97ac4fb235ef7c06451abf8272d88aa7130ee9/source/assets/stylesheets/_accessibility.scss#L69) background not display at all using `background: none`, rather than setting it to black.

Error report, run on [gov.uk](https://gov.uk):

![screen shot 2017-04-28 at 11 31 29](https://cloud.githubusercontent.com/assets/9826034/25525530/3b885976-2c07-11e7-939a-3e6de8fe392e.png)

**Comment by selfthinker** [comment]
This is one of those things that only theoretically fails but practically it doesn't, because the skip link is not visible until focussed. And when it's focussed, the colour contrast passes.

**Comment by timpaul** [comment]
Following the launch of the [GOV.UK Design System](https://design-system.service.gov.uk/), GOV.UK Template will now only get major bug fixes and security patches, so I'm going to close this.
